airflow/config_templates/config_unit_tests.yml

```diff
@@ -0,0 +1,221 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Defaults specified in this file replace the default values from
+# config.yml file during unit tests. If you want a value to have
+# different default during unit tests, specify it here.
+
+---
+
+core:
+  description: ~
+  options:
+    dags_folder:
+      description: |
+        The folder where your airflow pipelines live, most likely a
+        subfolder in a code repository. This path must be absolute.
+      version_added: ~
+      type: string
+      example: ~
+      default: "{AIRFLOW_HOME}/dags"
+    executor:
+      description: |
+        The executor class that airflow should use. Choices include
+        ``SequentialExecutor``, ``LocalExecutor``, ``CeleryExecutor``, ``DaskExecutor``,
+        ``KubernetesExecutor``, ``CeleryKubernetesExecutor`` or the
+        full import path to the class when using a custom executor.
+      version_added: ~
+      type: string
+      example: ~
+      default: "LocalExecutor"
+    dags_are_paused_at_creation:
+      description: |
+        Are DAGs paused by default at creation
```

Comment:
Absolutely the smallest of nitpicks, some Description fields end with punctuation, others don't.

Reply:
I will address those separately. I do not want to mix "real" change in config to make it easier for  review.